src/lib.rs

```diff
@@ -296,8 +358,9 @@ mod tests {
     #[test]
     fn test_pomodoro_reset_from_break() {
         // Given
-        let mut pomodoro = Pomodoro::new((0, 1), (0, 5));
-        pomodoro.start_or_pause();
+        
+        let sound = default_sound_path();
+        let mut pomodoro = Pomodoro::new((0, 3), (0, 2) ,sound, true);
```

Comment:
Important: this test fails because the times passed to the `Pomodoro` constructor were changed without updating the assertions checking their formatting. Reverting this to:
```suggestion
        let mut pomodoro = Pomodoro::new((0, 1), (0, 5) ,sound, true);
```
fixes it.

Reply:
Actually this isn't working with me not sure why.